### PR TITLE
chore(CHANGELOG): update to v9.2.0

### PR DESCRIPTION
**Pull request by barmac**
__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: packages/dmn-js/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,12 @@ All notable changes to [dmn-js](https://github.com/bpmn-io/dmn-js) are documente
 
 ___Note:__ Yet to be released changes appear here._
 
+## 9.2.0
+
+* `FEAT`: close select whenever there is user interaction outside of it ([#546](https://github.com/bpmn-io/dmn-js/issues/546), [#559](https://github.com/bpmn-io/dmn-js/issues/559))
```
Comment (MaxTru) [inline packages/dmn-js/CHANGELOG.md:11]: To make it easier to understand for the user maybe put: "close decision table hit-policy select"

File: packages/dmn-js/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,12 @@ All notable changes to [dmn-js](https://github.com/bpmn-io/dmn-js) are documente
...
+* `FEAT`: center resize column hitbox ([#554](https://github.com/bpmn-io/dmn-js/issues/554))
```
Comment (MaxTru) [inline packages/dmn-js/CHANGELOG.md:12]: As before: "center decision table resize colum hitbox"

**Comment by MaxTru** [comment]
Generally I think this is complete and can be merged. My comments are meant to be understood as improvements from my perspective, you decide.

File: packages/dmn-js/CHANGELOG.md (hunk 1 above)
Comment (barmac) [reply packages/dmn-js/CHANGELOG.md:11]: The feature changes behaviour of all selects in the library, so it's not only hit policy select.

**Pull request merged**